README.adoc

```diff
@@ -2,8 +2,7 @@
 
 Use this action to promote an image in Amazon Elastic Container Registry (Amazon ECR).
 
-Note: Information related to the artifacts created with this action will be published to CloudBees Platform.
-
```

Comment:
@Dawn-Davenport need to keep the blank line before the heading (== Inputs) to ensure proper rendering in markdown.